_targets.R

```diff
@@ -7,9 +7,35 @@ options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
 p1_targets_list <- list(
+  tar_target(
+    site_data_01427207,
+    download_nwis_site_data("01427207"),
+    format = "file"
+  ),
+  tar_target(
+    site_data_01432160,
+    download_nwis_site_data("01432160"),
+    format = "file"
+  ),
+  tar_target(
+    site_data_01435000,
+    download_nwis_site_data("01435000"),
+    format = "file"
+  ),
+  tar_target(
+    site_data_01436690,
+    download_nwis_site_data("01436690"),
+    format = "file"
+  ),
+  tar_target(
+    site_data_01466500,
+    download_nwis_site_data("01466500"),
+    format = "file"
+  ),
+  tar_target(tmp_data_dir, '1_fetch/tmp', format = "file"),
```

Comment:
See my comment responding to yours -- I think it best if we avoid this file directory target approach to track file targets that are generated dynamically by the pipeline and instead list those file targets directly as inputs to downstream targets.